content/en/docs/Getting started/install.md

```diff
@@ -106,12 +106,13 @@ For instance, it is possible to install cOS (or any derivative) with the install
 If in the rescue system, or LiveCD you have docker available, it can be used to perform an installation
 
 ```bash
-docker run --privileged -v $DEVICE:$DEVICE -ti quay.io/costoolkit/releases-green:cos-system-0.7.4-2 elemental install --docker-image $IMAGE $DEVICE
+docker run --privileged -v /dev/:/dev/ -ti quay.io/costoolkit/elemental:v0.0.14 install --docker-image $IMAGE $DEVICE
```

Comment:
just an idea, but probably we can use `quay.io/costoolkit/elemental:latest` here, so there are less chances we have it really outdated after some time.

Reply:
I was adamant to include the latest in here...I dont know if it works yet :D